beacon/validator/src/main/java/tech/pegasys/teku/validator/coordinator/performance/DefaultPerformanceTracker.java

```diff
@@ -205,7 +205,10 @@ private BlockPerformance getBlockPerformanceForEpochs(
 
     int numberOfProducedBlocks = producedBlocks.size();
     return new BlockPerformance(
-        numberOfBlockProductionAttempts, (int) numberOfIncludedBlocks, numberOfProducedBlocks);
+        endEpochExclusive,
```

Comment:
We're actually reporting performance for a range of epochs here whereas the log makes it look like its for a single epoch.  I'm not really sure why we're reporting over a range of epochs though, we should just report every epoch.

So this expands the scope of this change a bit, but I think it would be worth changing it to report block performance every epoch.  That's effectively setting `BLOCK_PERFORMANCE_EVALUATION_INTERVAL` to 1 but I think we can then remove the field and simplify the code so it no longer needs to support reporting over multiple epochs.